app/messages/Messages.scala

```diff
@@ -109,6 +109,10 @@ object Messages {
       english = "Set up a payment plan"
     )
 
+    val `Set up an Employers' PAYE payment plan`: Message = Message(
+      english = "Set up an Employers’ PAYE payment plan"
```

Comment:
Does this need to be one of those "smart quote" things? I'm not 100% sure so disregard if not, but I've been caught out by this in the past

Reply:
I made sure the message itself used a smart quote, but for the val name I chose a straight quote cos it's easier to type. Not that it's likely we're going to type the val name, but you never know...